.drone.star

```diff
@@ -12,7 +14,9 @@ def main(ctx):
     return checkStarlark() + \
            pnpmlint(ctx) + \
            unitTestPipeline(ctx) + \
-           e2eTests()
+           e2eTests()+ \
+           createArtifacts() + \
+           publishArtifacts()
```

Comment:
merging these two pipelines into two steps would be easier for artifacts sharing.
sharing between steps is easy than sharing between pipelines